scripts/installer

```diff
@@ -0,0 +1,287 @@
+#!/usr/bin/env bash
+
+# Copyright 2020 The Tekton Authors
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#     http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+OPENSHIFT="false"
+READONLY="false"
+UNINSTALL="false"
+INSTALL_NAMESPACE="tekton-pipelines"
+PIPELINES_NAMESPACE="tekton-pipelines"
+TRIGGERS_NAMESPACE="tekton-pipelines"
+
+# initOS discovers the operating system for this system.
+initOS() {
+  OS=$(echo `uname`|tr '[:upper:]' '[:lower:]')
+
+  case "$OS" in
+    # Minimalist GNU for Windows
+    mingw*) OS='windows';;
+  esac
+
+  echo $OS
+}
+
+# verifySupported checks that curl or wget and kubectl is available.
+verifySupported() {
+  if ! type "curl" > /dev/null && ! type "wget" > /dev/null; then
+    echo "Either curl or wget is required"
+    exit 1
+  fi
+
+  if ! type "kubectl" > /dev/null; then
+    echo "kubectl is required"
+    exit 1
+  fi
+
+  if ! type "sed" > /dev/null; then
+    echo "sed is required"
+    exit 1
+  fi
+}
+
+# checkDesiredVersion checks if the desired version is available.
+checkDesiredVersion() {
+  if [ "x$DESIRED_VERSION" == "x" ]; then
+    echo "No version specified, trying to find latest release ..."
+    # Get tag from release URL
+    local latest_release_url="https://github.com/tektoncd/dashboard/releases"
+    if type "curl" > /dev/null; then
+      TAG=$(curl -Ls $latest_release_url | grep 'href="/tektoncd/dashboard/releases/tag/v0.[0-9]*.[0-9]*\"' | grep -v no-underline | head -n 1 | cut -d '"' -f 2 | awk '{n=split($NF,a,"/");print a[n]}' | awk 'a !~ $0{print}; {a=$0}')
+    elif type "wget" > /dev/null; then
+      TAG=$(wget $latest_release_url -O - 2>&1 | grep 'href="/tektoncd/dashboard/releases/tag/v0.[0-9]*.[0-9]*\"' | grep -v no-underline | head -n 1 | cut -d '"' -f 2 | awk '{n=split($NF,a,"/");print a[n]}' | awk 'a !~ $0{print}; {a=$0}')
+    fi
+    echo "Release found: $TAG"
+  else
+    TAG=$DESIRED_VERSION
+  fi
+}
+
+# downloadFile downloads the desired manifest.
+downloadFile() {
+  MANIFEST="tekton-dashboard-release"
+  if [ "$OPENSHIFT" == "true" ]; then
+    MANIFEST="openshift-$MANIFEST"
+  fi
+  if [ "$READONLY" == "true" ]; then
+    MANIFEST="$MANIFEST-readonly"
+  fi
+  MANIFEST="$MANIFEST.yaml"
+  DOWNLOAD_URL="https://github.com/tektoncd/dashboard/releases/download/v0.7.0/$MANIFEST"
```

Comment:
hardcoded version here, shouldn't it be using $TAG?

Reply:
definitely !

Reply:
fixed